bin.src/processFile.py

```diff
@@ -140,8 +143,8 @@ def run(config, inputFiles, weightFiles=None, varianceFiles=None,
         # process the data
         #
         if config.doCalibrate:
-            result = calibrateTask.run(exposure)
-            exposure, calibSources = result.exposure, result.sources
+            result = charImageTask.characterize(exposure)
+            exposure, calibSources = result.exposure, result.sourceCat
         else:
             calibSources = None
             if not exposure.getPsf():
```

Comment:
The line just after this (Line 151) needs to be updated to `charImageTask`

```
            if not exposure.getPsf():
                calibrateTask.installInitialPsf(exposure)
```

should presumably be

```
            if not exposure.getPsf():
                charImageTask.installInitialPsf(exposure)
```